_data/blogs.yml

```diff
@@ -267,3 +267,10 @@
   title: Here I come Coding In
   img: haider2.jpg
   author_name: Haider Shafaq
+
+- url: https://technoorja.wordpress.com/2018/01/01/technoorja-learns-to-code/
+  title: Technoorja Learns To Code
+  img: Oorja_modified.jpg
```

Comment:
Please upload this image in the img/blogs folder

Reply:
I have uploaded that image